benchmark-overhead-jmh/build.gradle.kts

```diff
@@ -28,20 +28,37 @@ tasks {
   val jmhFork = gradle.startParameter.projectProperties["jmh.fork"]?.toInt()
   val jmhIncludes = gradle.startParameter.projectProperties["jmh.includes"]
 
+  // this option should point to the benchmark that you want to run with JFR
```

Comment:
Is there a reason to not keep this boolean? It's a bit mysterious to have this and jmh.includes both adding to the included benchmarks. Or should jmh.includes be ignored when flightRecording is set because there's some requirement only a single benchmark is run?

Reply:
ya, it was not clear, I generalized the parameters, removed the duplication, and added some comments